packages/tea-map-cache/tests/test-tea.js

```diff
@@ -4,27 +4,33 @@ const test = require('ava');
 const proxyquire = require('proxyquire').noPreserveCache();
 
 test('getTeaBucketPath returns a mapped bucket path on an expected response from TEA', async (t) => {
-  const { getTeaBucketPath } = proxyquire('../tea', {
+  const { getTeaBucketPath } = proxyquire('../dist/tea', {
     got: {
       get: async () => ({ body: '["fake-bucket-redirect/path"]' })
     }
   });
 
-  const actual = await getTeaBucketPath();
+  const actual = await getTeaBucketPath({
```

Comment:
So we do `undefined` for the params to match the type expectations and because we're mocking the real response anyway?

Reply:
Would there be harm in providing real values to these parameters? The test is a little confusing to me as is

Reply:
No harm at all, just due to the typescript compilation we can't shenanigans it with no values at all.     undefined feels a bit more explicit in that it makes it obvious it's not a normal call, but a string 'unusedTestValue' or something like that seems fine here.

Reply:
how about 21b6e8c45